### PR TITLE
updates caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @gusshawstewart
- [ ] @ztsygan
- [ ] @lpoolma1
- [ ] @bseeger
- [ ] @jhujasonw
- [ ] @reinacmurray
- [ ] @derekbelrose
- [ ] @jgara
- [ ] @jaredgalanis
- [ ] @Bonni
- [ ] @jwang40
- [ ] @akatz2
- [ ] @EvanBolt
- [ ] @cyork
- [ ] @jhu-alistair
- [ ] @michaelsainz
- [ ] @acornwe3
- [ ] @parkerbxyz

### PR DESCRIPTION
**Pull request by bseeger**
TL;DR. Adding a new caption to my post. 

### Done
- closes  #2 
- created a PR

### Todo
- [ ] merge PR
- [ ] delete branch
- [x] make lunch

cc: @jhujasonw

### CODE REVIEW

File: _posts/0000-01-04-bseeger.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Mooooo!  Aren't cows great?
```
Comment (gusshawstewart) [inline _posts/0000-01-04-bseeger.md:6]: this is really good, but what about chickens?

Comment (gusshawstewart) [inline _posts/0000-01-04-bseeger.md:6]:
```suggestion
Cluck aren't chickens better?
```

Comment (gusshawstewart) [reply _posts/0000-01-04-bseeger.md:6]: 👍

Comment (parkerbxyz) [reply _posts/0000-01-04-bseeger.md:6]:
```suggestion
 _________ 
< Mooooo! >
 --------- 
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

Aren't cows great? 
```

File: _posts/0000-01-04-bseeger.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Mooooo!  Aren't cows great? 
+
```
Comment (jhujasonw) [inline _posts/0000-01-04-bseeger.md:7]:
```suggestion
![morechicken](https://skylerguns.files.wordpress.com/2016/06/chickfila01lg.jpg?w=1140)
```

**Comment by parkerbxyz** [comment]
@bseeger I have a feeling you'd be a fan of `cowsay` if you aren't already: https://en.wikipedia.org/wiki/Cowsay

**Comment by bseeger** [comment]
Thanks @parkerbxyz  - that's funny!

**Pull request merged**